wazo_call_logd/database/dao.py

```diff
@@ -149,6 +157,17 @@ def _apply_filters(self, query, params):
                        for user_uuid in params['user_uuids'])
             query = query.filter(sql.or_(*filters))
 
+        if params.get('tenant_uuids'):
+            filters = [CallLogSchema.participant_tenant_uuids.contains(str(tenant_uuid))
+                       for tenant_uuid in params['tenant_uuids']]
+            filters.extend([
+                CallLogSchema.requested_tenant_uuid.in_(params['tenant_uuids']),
+                CallLogSchema.requested_internal_tenant_uuid.in_(params['tenant_uuids']),
+                CallLogSchema.source_internal_tenant_uuid.in_(params['tenant_uuids']),
+                CallLogSchema.destination_internal_tenant_uuid.in_(params['tenant_uuids']),
+            ])
```

Comment:
As mentioned above, I think we should only search through the `tenant_uuid` of the call log (not participant)